Nightly failure on phpstan-php7.4 pipeline

Nightly Failing

```bash
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo controller db service

------ -------------------------------------------------------------------- 
  Line   db/note.php                                                         
 ------ -------------------------------------------------------------------- 
         Ignored error pattern #Unsafe usage of new static\(\).# in path     
         /var/www/owncloud/server/apps/notes/db/note.php was not matched in  
         reported errors.                                                    
 ------ -------------------------------------------------------------------- 


 [ERROR] Found 1 error                                                          

make: *** [Makefile:215: test-php-phpstan] Error 1
```

Builds:
https://drone.owncloud.com/owncloud/notes/1923/2/4
https://drone.owncloud.com/owncloud/notes/1922/2/4
https://drone.owncloud.com/owncloud/notes/1921/2/4

--- a/phpstan.neon
+++ b/phpstan.neon
@@ -4,10 +4,6 @@ parameters:
     - %currentWorkingDirectory%/../../lib/base.php
   ignoreErrors:
     -
-      message: '#Unsafe usage of new static\(\).#'
-      path: db/note.php
-      count: 1
-    -
       message: '#Strict comparison using === between string and null will always evaluate to false.#'
       path: service/notesservice.php
       count: 1

